community/modules/scheduler/schedmd-slurm-gcp-v6-controller/etc/htc-slurm.conf.tpl

```diff
@@ -0,0 +1,67 @@
+# slurm.conf
+# https://slurm.schedmd.com/high_throughput.html
+
+ProctrackType=proctrack/cgroup
+SlurmctldPidFile=/var/run/slurm/slurmctld.pid
+SlurmdPidFile=/var/run/slurm/slurmd.pid
+TaskPlugin=task/affinity,task/cgroup
+MaxArraySize=10001
+MaxJobCount=500000
+MaxNodeCount=100000
+MinJobAge=60
```

Comment:
Can you confirm that these actually make it into the deployed cluster. I believe they would be under /etc/slurm/slurm.conf. Same for db settings. For example, you should see `MaxArraySize=10001` in the deployed cluster.

Reply:
```
[root@htcslurmv6-controller slurm]# cat slurm.conf 
# slurm.conf
# https://slurm.schedmd.com/high_throughput.html

ProctrackType=proctrack/cgroup
SlurmctldPidFile=/var/run/slurm/slurmctld.pid
SlurmdPidFile=/var/run/slurm/slurmd.pid
TaskPlugin=task/affinity,task/cgroup
MaxArraySize=10001
MaxJobCount=500000
MaxNodeCount=100000
MinJobAge=60
```

Reply:
For slurmdbd
```
root@htcslurmv6-controller slurm]# cat slurmdbd.conf 
# slurmdbd.conf
# https://slurm.schedmd.com/slurmdbd.conf.html

DebugLevel=info
PidFile=/var/run/slurm/slurmdbd.pid

# https://slurm.schedmd.com/slurmdbd.conf.html#OPT_CommitDelay
CommitDelay=1

################################################################################
#              vvvvv  WARNING: DO NOT MODIFY SECTION BELOW  vvvvv              #
################################################################################

AuthType=auth/munge
AuthAltTypes=auth/jwt
AuthAltParameters=jwt_key=/var/spool/slurm/jwt_hs256.key

DbdHost=htcslurmv6-controller

LogFile=/var/log/slurm/slurmdbd.log

SlurmUser=slurm

StorageLoc=slurm_acct_db

StorageType=accounting_storage/mysql
StorageHost=localhost
StoragePort=3306
StorageUser=slurm
StoragePass=""

################################################################################
#              ^^^^^  WARNING: DO NOT MODIFY SECTION ABOVE  ^^^^^              #
################################################################################
```